tests/Mocks/Helper/RabbitMQMessageHelper.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gamee\RabbitMQ\Tests\Mocks\Helper;
+
+final class RabbitMQMessageHelper
+{
+
+	/**
+	 * @var RabbitMQMessageHelper
+	 */
+	private static $self;
+
+	/**
+	 * @var string[][]
+	 */
+	private $queueMessages = [];
+
+	/**
+	 * @var array
+	 */
+	private $config;
+
+
+	private function __construct(array $config)
+	{
+		self::$self = $this;
+		$this->config = $config;
+	}
+
+
+	public static function getInstance(array $config = null): self
+	{
+		if (self::$self === null) {
+			if ($config === null) {
+				throw new \Exception('Missing config');
+			}
+			self::$self = new RabbitMQMessageHelper($config);
+		}
+
+		return self::$self;
+	}
+
+
+	public function reinit(): void
```

Comment:
What is the purpose of this method?

Reply:
to make sure the to get clear mock without any messages inside